src/main/webapp/app/exercises/shared/rating/rating.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+import { Rating } from 'app/entities/rating.model';
+import { SERVER_API_URL } from 'app/app.constants';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class RatingService {
+    private ratingResourceUrl = SERVER_API_URL + 'api/results/';
+
+    constructor(private http: HttpClient) {}
+
+    /**
+     * Update the student rating for feedback on the server.
```

Comment:
I think the word `update` should be avoided here since it could be misinterpreted as: The rating already exists and we want to update it (even though this is not the case here, it is being created).